docker/airflow/dags/monkeypox_dag.py

```diff
@@ -0,0 +1,181 @@
+    load_cases_kwargs : dict, optional
+        dictionary of kwargs for load_cases. The default is {}.
+    to_parquet_kwargs : dict, optional
+        dictionary of kwargs for to_parquet. The default is {}.
+    read_parquet_kwargs : dict, optional
+        dictionary of kwargs for read_parquet. The default is {}.
+    conf_aggr : dict, optional
+        acts as group_and_aggr(raw_data, **conf_aggr). 
+        Default is:
+            {'cases': dict(column=csv_specs.countrycol, date_col=csv_specs.confdatecol,
+                            dropna=True, entrytype='cases', dropzeros=True),
+             'deaths' : dict(column=csv_specs.countrycol, date_col=csv_specs.deathdatecol,
+                             dropna=True, entrytype='deaths', dropzeros=True)}
+    sus_aggr : dict, optional
+        analogous to conf_aggr but acts on suspected entries. Default is:
+            {'sus_cases' : dict(column=csv_specs.countrycol, date_col=csv_specs.entrydatecol,
+                            dropna=True, entrytype='cases', dropzeros=True),
+            'sus_deaths' : dict(column=csv_specs.countrycol,
+                                             date_col=csv_specs.deathdatecol, 
+                                             dropna=True, entrytype='deaths', dropzeros=True)}
+        
+
+    Raises
+    ------
+    ValueError
+        No data to aggregate if rawfile results empty or not available.
+    
+    Note
+    ----
+    Do not use the same keys conf_aggr and sus_aggr, otherwise your files for suspected entries will overwrite those with confirmed ones
+
+    Returns
+    -------
+    None
+
+    """
+
+    @task
+    def extract(rawfile: str, folder, load_cases_kwargs: dict = {},
+                to_parquet_kwargs: dict = {}):
+        """
+        Parameters
+        ----------
+        rawfile : str
+            DESCRIPTION.
+        folder : str
+            Location for output files.
+        load_cases_kwargs : dict, optional
+            DESCRIPTION. The default is {}.
+        to_parquet_kwargs : dict, optional
+            DESCRIPTION. The default is {}.
+
+        Returns
+        -------
+        path : str
+            The path where rawfile has been copied in a parquet format
+
+        """
+        try:
+            raw_data = load_cases(rawfile, **load_cases_kwargs)
+            logging.info('raw data loaded')
+        except Exception as e:
+            tback = traceback.format_exc()
+            logging.critical(f'loading raw data failed. Error message and traceback below.\n {e} \n {tback}')
+        date = pd.to_datetime('today').strftime('%d_%m_%Y')
+        path = os.path.join(folder, f'{date}.parquet')
+        raw_data.to_parquet(path, **to_parquet_kwargs)
+        logging.info(f'raw data saved in {path}')
+        return path
+
+    @task
+    def transform_and_load(path: str, folder: str, read_parquet_kwargs: dict = {},
+                           conf_aggr: dict = {}, sus_aggr: dict = {}):
+        """
```

Comment:
Before the list of parameters add a short but informative description of the purpose of this function

Reply:
Done!